src/SchemaMigrations/DefinedSchemas.js

```diff
@@ -171,9 +173,8 @@ export class DefinedSchemas {
   // Create a fake session since Parse do not create the _Session until
   // a session is created
   async createDeleteSession() {
-    const session = new Parse.Session();
-    await session.save(null, { useMasterKey: true });
-    await session.destroy({ useMasterKey: true });
+    const { response } = await rest.create(this.config, Auth.master(this.config), '_Session', {});
+    await rest.del(this.config, Auth.master(this.config), '_Session', response.objectId);
   }
```

Comment:
Owww I see here @dblythy  that you encountered the error since the SDK uses the port.

Reply:
the old code should not be changed, if the old code works, any Parse SDK/GraphQL/REST call during the init will work.